ibis/expr/types/numeric.py

```diff
@@ -1042,7 +1042,7 @@ def histogram(
             if base is None:
                 base = self.min().over(empty_window) - eps
```

Comment:
Why do we need this window here? can't we do a simple min() and max()?

Reply:
Made these changes. Great suggestion, cleaned up more code than I was expecting!